test/cpp/test_status_common.h

```diff
@@ -77,36 +78,20 @@ class StatusTest : public testing::TestWithParam<CppStacktracesMode> {
 
 namespace testing {
 
+#define THROW_RUNTIME_ERROR_FROM_C10_ERROR(block) \
```

Comment:
Can we prefix our macros with `XLA_` to:

- avoid potential clashes with other libraries,
- make it clear that they are our own macros
?